### ISSUE TITLE
Tenant Onboarding fails with CPV Refresh (Tenant not found)

### ISSUE THREAD

**Issue by support-eworx-at** [KelvinTegelaar/CIPP#2474]
### Description

Tenant Onboarding fails with CPV Refresh (Tenant not found). Worked fine with Backend Version 5.7.0, startet with Version 5.7.2. Tested multiple tenants - issue occures with every tenant.

Screenshot of error:
![2024-05-23 09_51_57-image (1)](https://github.com/KelvinTegelaar/CIPP/assets/147516094/575ec8f7-9bb4-4093-b820-3d940227b2c0)

### Environment data

```PowerShell
Sponsored instance Front end version number: 5.7.0 
Back end version number: 5.7.2
Tried Tenant Cache Clear: true
Tried Token Cache Clear: false -> where? in KB Page is not found?? https://docs.cipp.app/setup/installation/cleartokencache
```

**Comment by support-eworx-at** [comment]
I confirm that I have checked the documentation thoroughly and believe this to be an actual bug.

**Comment by snurker** [comment]
I am having similar issues.

**Comment by Sahasrahla** [comment]
We setup a brand new, very first instance of CIPP today, and are also receiving this error. We assumed it was due to improper configuration on our part until Google lead us here.

**Comment by JoshBritton5544** [comment]
We are seeing the same behaviors on new onboarding attempts after the front-end update to 5.7.0. We attempted to run the CPV refresh from the Tenant settings and that appears to work as expected, but the refresh does not complete on the Tenant Onboarding Wizard.

**Comment by snurker** [comment]
We were able to get our tenants added by manually doing the refresh as well

Matthew Snurkowski
Peach State Technology<http://www.peachstatetechnology.com/>
Phone: 404-429-5250

Please excuse any typos or grammatical errors.  Sent from mobile device.
________________________________

**Comment by wstanley1977** [comment]
We are seeing this issue too

**Comment by support-eworx-at** [comment]
-> Backend updated to 5.7.4, no change.

**Comment by zanderson-aim** [comment]
Yep hit this as well, adding note to follow the issue

**Comment by KelvinTegelaar** [comment]
Fixed in dev, temporary workaround is doing a manual CPV refresh via Application Settings -> Tenants.

**Comment by support-eworx-at** [comment]
Hi @KelvinTegelaar, wrote a mail to helpdesk before an hour. CPV refresh does not works as a workaround in our environment.
BR, Daniel

**Comment by jscooksey** [comment]
Having the same issue as well since 5.7.0 and 5.7.2

**Comment by support-eworx-at** [comment]
Seems to be fixed with Frontend 5.7.1 & Backend 5.7.4

**Comment by jscooksey** [comment]
Yep, cleaned up the last one remaining, and Ive been able to onboard new tenants without issue since.  Although thats only been 2 so far
